cmd/really/testLogger.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "testLogger.h"
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+
+void testLogger::log(qtransport::LogLevel level, const std::string &string) {
+  const char *lvl;
+  switch (level) {
+  case qtransport::LogLevel::fatal:
```

Comment:
Just for consistency with syslog, I called this "critical" in the Cantina logger.  I'd suggest the same here for that same reason.

Reply:
We can refactor that to match, but that will need to first be done under transport. Let's do that in a different PR considering that may involve several parent modules that use transport.